client/my-sites/plugins/plugin-details.jsx

```diff
@@ -366,7 +367,11 @@ function PluginDetails( props ) {
 			/>
 			<QueryPlugins siteId={ selectedSite?.ID } />
 			<QueryEligibility siteId={ selectedSite?.ID } />
-			<QuerySiteFeatures siteIds={ selectedOrAllSites.map( ( site ) => site.ID ) } />
+			{ selectedOrAllSites && 1 === selectedOrAllSites.length ? (
+				<QuerySiteFeatures siteIds={ selectedOrAllSites.map( ( site ) => site.ID ) } />
+			) : (
+				<QueryJetpackSitesFeatures />
+			) }
```

Comment:
Here, we're using the Jetpack bulk request instead of one request for each site.